tracing-subscriber/src/fmt/mod.rs

```diff
@@ -476,6 +476,7 @@ where
         subscribe::Subscribe<Registry> + Send + Sync + 'static,
 {
     /// Finish the builder, returning a new `FmtCollector`.
+    #[must_use = "You may want to use `try_init` or similar to actually install the collector."]
```

Comment:
nit: normally these are lower-case
```suggestion
    #[must_use = "you may want to use `try_init` or similar to actually install the collector."]
```